### ISSUE TITLE
Lost access to the web interface DFP-34X-2C2

### ISSUE THREAD

**Issue by Karetov** [Anime4000/RTL960x#46]
Hello

After saving the settings on my stick, I can't get into the web interface, the verse does not respond to the assigned address. 
Either I accidentally disabled the DHCP server or it was due to two identical VLAN IDs. 
I want to flash the chip using a programmer, but there are no suitable firmware files, please tell me, can you help me?

**Comment by Anime4000** [comment]
have you access to telnet?

**Comment by Karetov** [comment]
I tried to connect via telnet, but it didn't work out, maybe the stick has a changed ip address or it's due to another problem

**Comment by Anime4000** [comment]
I see, before you flash the SPI, try access via UART and Factory Reset by erasing mtd3 partition

**Comment by Karetov** [comment]
Ihave absolutely no experience in UART and only have a TL866CL programmer. I don't even know where to connect on the stick board, except to the MX chip 

If you find time to describe the process in more detail, I would be very grateful. Or share links to information about what I need.

**Comment by Anime4000** [comment]
UART on ODI stick is very easy, you just need UART to USB and solder 2 wire `RX`, `TX` and attach `Ground` to router body,

I have guide and location of UART pads here:
https://github.com/Anime4000/RTL960x/blob/main/Docs/UART.md#rtl9601d

**Comment by Karetov** [comment]
Thank you so much for your help and prompt response! I'll try to do

**Comment by Karetov** [comment]
Purchased a USB to UART CH340G. 
I have a couple of questions.
Should GND be attached to the SFP enclosure?

Do I need to insert the SFP into the router port after that? 
If you need to insert it into the router, then do you need 3.3v power? If so, where to submit it?
In the examples in the photo I saw a soldered wire, do I need to solder the wire to the chip leg for RTL9601D?

Thank you in advance for your help

**Comment by Anime4000** [comment]
> Should GND be attached to the SFP enclosure?

Yes, any ground, SFP Cage, Router metal body...

> Do I need to insert the SFP into the router port after that?

After attach RX-TX, TX-RX, GND-Router GND, insert SFP into SFP Slot slowly, make sure RTL9601D is facing up/top then align and insert

> If you need to insert it into the router, then do you need 3.3v power? If so, where to connect it?

No need 3.3v from UART because SFP Stick using a lot of power, UART has not enough to supply higher Amp

**Comment by Karetov** [comment]
I want to thank you for your help, everything worked out. The connection turned out to be much easier than I thought and than it was described in the manual.

I will keep an eye on updates)
I ran into a problem on the provider side. After connecting via DHCP to the module in bridge mode, my router could not get an IP address, it was necessary to create a PPPoE, enter a random username and password and after an error message switch back to DHCP, after which I received an IP address. 

Perhaps such a setting with a one-time request for PPPoE can be provided in the firmware. 

Good luck with the project!

**Comment by Karetov** [comment]
I ran into a problem again when I can't log in to the stick.. It happens if you update the verse to the hybrid firmware, add VAN10 there, which is added to the stock firmware when the vendor is specified. 

After installing the stock firmware, I got 2 VAN10, one of which is editable in bridge mode (created by me), the second is non-editable. 

After that, I accidentally deleted VAN30 for the Internet, as a result, I had two VAN10S left, after which I could not go to the verse anymore

By UART, the process does not reach the login and password request..

**Comment by Karetov** [comment]
I ran into a problem again when I can't log in to the stick.. It happens if you update the verse to the hybrid firmware, add VAN10 there, which is added to the stock firmware when the vendor is specified. 

After installing the stock firmware, I got 2 VAN10, one of which is editable in bridge mode (created by me), the second is non-editable. 

After that, I accidentally deleted VAN30 for the Internet, as a result, I had two VAN10S left, after which I could not go to the verse anymore

By UART, the process does not reach the login and password request..

**Comment by Karetov** [comment]
[Uploading Putty.zip…]()

**Comment by StavLex** [comment]
Hello. Soldered the uart to the pads. I see a log in telnet on a com port, but it does not allow enter commands. what to do? Once it turned out to connect, backup the config folder and deleted everything from the original except the .date and .dat, bat dont look web-interface of stick. Firmware M110_sfp_ODI_210702.tar. My mikrotik see sfp and the web interface of the stick was available until I check the box Enable IGMP PROXY.

**Comment by d0bro** [comment]
Hello. I buy 2 ps ODI Realtek DFP-34G-2C2. I Have Mikrotik, And I couldn't access the web interface. 
I tried different subnets. I disabled automatic speed reduction and set 1GB BaseT full
Configured networks and masquerading (I'm not a newbie)
Tried to shove it into a server with a network SFP PCI card, and configure the interface manually.

![4qzta _09-17_14-21-28](https://github.com/user-attachments/assets/879b0cbc-25ab-4642-ac98-9d800632d476)

I disassembled the stick, soldered the UART and moved the slot from 0 to 1 (setenv sw_active 1 and setenv sw_commit 1) Without result. How can I revive it? What commands to enter in the tty terminal?

Another question: 
What do the ip address and ip address_server parameters mean in the variables?
